### ISSUE TITLE
Reporting example for election cycle dates 

### ISSUE THREAD

**Issue by kathycarothers** [fecgov/fec-cms#5036]
### Summary

**What we're after:**
_We need to make it easier for the reporting community to report the election cycle to date on their second Q report. In order to make it easier for the community to understand we plan to provide them with a reporting example and place it on two of our quarterly report web pages. This will then be included in the global search so when folks do a search it will come up too._

### Related issues

_List any relevant related issue(s)_

- #5037
- #4880

### Completion criteria

- [x] RAD will draft the reporting example
- [x] Incorporate RAD feedback
- [x] Content team review
- [x] Once approved draft the reporting example in Wagtail - this will be a child page from this page https://www.fec.gov/help-candidates-and-committees/reporting-examples/, add to page too as a link 
- [ ] Add button or link to https://www.fec.gov/help-candidates-and-committees/filing-reports/quarterly-reports/, and https://www.fec.gov/help-candidates-and-committees/dates-and-deadlines/2022-reporting-dates/2022-quarterly-filers/

### Future work

_Add to best bet for the global search_

**Comment by bmathesonFEC** [comment]
Distributed the draft reporting example to RAD. Waiting on feedback. Next step, distribute among the Content Team.

**Comment by bmathesonFEC** [comment]
Incorporated RAD edits, distributed to the Content team for review.

**Comment by bmathesonFEC** [comment]
Edits made, page published, links added.

A few notes about the decisions made for this page that were made before closing this ticket:
- instead of having a bullet text list for the cycles/dates I used an HTML block to create a table
- Originally I planned to include example images without captions, but caption text is a required field for a reporting example image. I kept it simple in those situations.
- Cannot add a reporting example card to the 2022 Quarterly Reports page because the place where I would add it is in the "introduction" section of the resource template. Added a link to the reporting example in the READ NEXT sidebar:
https://www.fec.gov/admin/pages/11934/edit/preview/
- Added a reporting example card to the candidate guide quarterly reporting page:
https://www.fec.gov/help-candidates-and-committees/filing-reports/quarterly-reports/
